src/main/drivers/dshot_bitbang.h

```diff
@@ -20,13 +20,16 @@
 
 #pragma once
 
+#include "platform.h"
+
+#include "common/time.h"
+
+#include "drivers/dma.h"
+#include "drivers/io_types.h"
+#include "drivers/motor_types.h"
```

Comment:
.h file should ideally `#include` only headers that are directly needed. 
In this case  pg/motor.h should probably pull necessary dependencies. But sometimes it leads to ugly cycles or too many dependencies.